### ISSUE TITLE
uninitialized constant CircleCi::Request::OpenSSL (NameError) Did you mean?  OpenStruct

### ISSUE THREAD

**Issue by hugoduncan** [circle-cli/circle-cli#21]
```
$ gem install circle-cli
Fetching: circleci-1.0.1.gem (100%)
Successfully installed circleci-1.0.1
Fetching: rugged-0.24.0.gem (100%)
Building native extensions.  This could take a while...
Successfully installed rugged-0.24.0
Fetching: gitable-0.3.1.gem (100%)
Successfully installed gitable-0.3.1
Fetching: circle-cli-0.1.1.gem (100%)
Successfully installed circle-cli-0.1.1
4 gems installed
```

```
$ circle
/Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circleci-1.0.1/lib/circleci/request.rb:11:in `<class:Request>': uninitialized constant CircleCi::Request::OpenSSL (NameError)
Did you mean?  OpenStruct
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circleci-1.0.1/lib/circleci/request.rb:7:in `<module:CircleCi>'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circleci-1.0.1/lib/circleci/request.rb:2:in `<top (required)>'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circleci-1.0.1/lib/circleci.rb:16:in `require_relative'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circleci-1.0.1/lib/circleci.rb:16:in `block in <top (required)>'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circleci-1.0.1/lib/circleci.rb:16:in `each'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circleci-1.0.1/lib/circleci.rb:16:in `<top (required)>'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circle-cli-0.1.1/lib/circle/cli/project.rb:1:in `<top (required)>'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circle-cli-0.1.1/lib/circle/cli/app.rb:3:in `<top (required)>'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circle-cli-0.1.1/lib/circle/cli.rb:3:in `<top (required)>'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circle-cli-0.1.1/lib/circle-cli.rb:1:in `<top (required)>'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circle-cli-0.1.1/bin/circle:3:in `<top (required)>'
    from /Users/duncan/.rbenv/versions/2.3.1/bin/circle:23:in `load'
    from /Users/duncan/.rbenv/versions/2.3.1/bin/circle:23:in `<main>'
```

**Comment by rlester** [comment]
Seeing the same issue on a clean install.

**Comment by rzane** [comment]
This seems to be a bug in the latest version of the circleci gem. Issue reported here: https://github.com/mtchavez/circleci/pull/81.

**Comment by hugoduncan** [comment]
Seems like the fix in mtchavez/circleci#81 has led to another issue:

```
$ circle
/Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circleci-1.0.2/lib/circleci/response.rb:7:in `<class:Response>': uninitialized constant CircleCi::Response::Forwardable (NameError)
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circleci-1.0.2/lib/circleci/response.rb:6:in `<module:CircleCi>'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circleci-1.0.2/lib/circleci/response.rb:2:in `<top (required)>'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circleci-1.0.2/lib/circleci.rb:16:in `require_relative'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circleci-1.0.2/lib/circleci.rb:16:in `block in <top (required)>'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circleci-1.0.2/lib/circleci.rb:16:in `each'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circleci-1.0.2/lib/circleci.rb:16:in `<top (required)>'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circle-cli-0.1.1/lib/circle/cli/project.rb:1:in `<top (required)>'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circle-cli-0.1.1/lib/circle/cli/app.rb:3:in `<top (required)>'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circle-cli-0.1.1/lib/circle/cli.rb:3:in `<top (required)>'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circle-cli-0.1.1/lib/circle-cli.rb:1:in `<top (required)>'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
    from /Users/duncan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/circle-cli-0.1.1/bin/circle:3:in `<top (required)>'
    from /Users/duncan/.rbenv/versions/2.3.1/bin/circle:23:in `load'
    from /Users/duncan/.rbenv/versions/2.3.1/bin/circle:23:in `<main>'
```

**Comment by codeurge** [comment]
@hugoduncan I believe this issue is resolved now with the new version of the CircleCI gem.
